cmd/fs-v1.go

```diff
@@ -42,7 +42,8 @@ import (
 type FSObjects struct {
 	// Path to be exported over S3 API.
 	fsPath string
-
+	// meta json file
```

Comment:
Need more explanation here why this file name is made configurable.

Reply:
done.